app/code/Magento/AsynchronousOperations/Controller/Adminhtml/Notification/Dismiss.php

```diff
@@ -6,44 +6,36 @@
 namespace Magento\AsynchronousOperations\Controller\Adminhtml\Notification;
 
 use Magento\AsynchronousOperations\Model\BulkNotificationManagement;
-use Magento\Backend\App\Action\Context;
 use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\Controller\Result\Json;
 use Magento\Framework\Controller\ResultFactory;
 
 /**
  * Class Bulk Notification Dismiss Controller
  */
-class Dismiss extends Action
+class Dismiss extends Action implements HttpGetActionInterface
```

Comment:
I assume using of Abstract action is deprecated. Make sence to refactor that? Or we will leave it as it is now?

Reply:
I think such refactoring would be excessive for this PR.